src/columns.js

```diff
@@ -200,23 +205,27 @@ export class Columns {
             th.innerHTML = ""
         }
 
-        this.dt.headings.push(th)
+        // this.dt.headings.push(th)
```

Comment:
remove this line entirely?

Reply:
no.  We will need to update the headings, but not by modifying the `headings` property itself, which is now a getter (a function that retrieves the current value).  I believe the line to replace that would be 
```javascript
this.dt.thead.appenChild()
```